src/toplevel_generators.jl

```diff
@@ -33,9 +33,9 @@ function generate_imports(src_name)
             HostPlatform() = platform_key_abi()
         end
     else
-        # Use fast stdlib-based Artifacts
+        # Use fast stdlib-based Artifacts + Preferences
         return quote
-            using Libdl, Artifacts, Base.BinaryPlatforms
+            using Libdl, Artifacts, Preferences, Base.BinaryPlatforms
```

Comment:
This makes it hard for us to generate the JLL wrappers in BinaryBuilder, as there are different dependencies for different versions of Julia

Reply:
To close the loop on this; I think we will simply always declare a dependence on `Preferences.jl` from BB; we just won't load it.